draft-ietf-acme-acme.md

```diff
@@ -482,7 +482,7 @@ to errors, this document defines the following standard tokens for use in the
 | invalidContact        | The contact URI for a registration was invalid     |
 | rejectedIdentifier    | The server will not issue for the identifier       |
 | unsupportedIdentifier | Identifier is not supported, but may be in future  |
-| agreementRequired     | The client must agree to terms before proceeding   |
+| userActionRequired    | The user must visit a URL before proceeding        |
```

Comment:
I think changing "The user must visit a URL before proceeding" to "The user must perform an action specified by a URL before proceeding" would clarify this. As-is one might get the impression that a GET to the URL is sufficient and I think we're imagining something more involved.